answerking_app/models/serializers.py

```diff
@@ -298,17 +305,95 @@ def validate_lineItems(self, line_items_data):
 
     class Meta:
         model = Order
-        fields = (
+        fields = [
             "id",
             "createdOn",
             "lastUpdated",
             "orderStatus",
             "orderTotal",
             "lineItems",
-        )
+            "owner",
+        ]
         depth = 3
 
 
+class ManagerAuthSerializer(serializers.ModelSerializer):
```

Comment:
If we are going with the permission as "IsStaffUser" we may want to change this to StaffAuthSerializer. Similar with a few other naming conventions further down also.